.appveyor.yml

```diff
@@ -4,11 +4,13 @@ environment:
   GOPATH: c:\gopath
   ELVISH_TEST_TIME_SCALE: 10
 install:
-  # Needed since the codecov uploader depends on a UNIX "find" command.
+  # The coverage script and codecov uploader depend on a POSIX shell and
+  # commands such as `find` so ensure the relevant Windows MSYS2 directory is
+  # in the external command search path.
```

Comment:
This comment no longer needs to be updated, revert this chunk.

Reply:
I would argue my comment better reflects reality. This Appveyor config doesn't just depend on a `find` command. It explicitly runs the `bash` command to execute a bash script.

Reply:
That makes sense, but please remove the reference to the now non-existing "coverage script" from the comment.

Reply:
Done